tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -368,4 +371,37 @@ INSTANTIATE_TEST_SUITE_P(
         testing::Values(Sizes({n, 1})),
         testing::Values(Sizes({n}))));
 
+using MatmulNodeTest = NVFuserTest;
+
+TEST_F(MatmulNodeTest, OutputStrides) {
```

Comment:
Any reason for requiring a C++ test?
We already have https://github.com/NVIDIA/Fuser/blob/0e823fe2f977556b762643e0b1c2a193322e2c7c/tests/python/direct/test_matmul.py#L204-L223

Reply:
https://teams.microsoft.com/l/message/19:7f200e6458394d3481c2b1cba6fe9e61@thread.tacv2/1758850541504?tenantId=43083d15-7273-40c1-b7db-39efd9ccc17a&groupId=74537292-c203-4a6d-aea0-40c527b969f7&parentMessageId=1758849956300&teamName=csarofeen-staff-and-guests&channelName=nvFuser&createdTime=1758850541504

Reply:
Added this to the PR description as well